tests-examples/demo-todo-app.spec copy.js

```diff
@@ -0,0 +1,293 @@
+// @ts-check
```

Comment:
Are there any differences with the `demo-todo-app.spec.js` file? Can we live just one file?